ci/windows/run.ps1

```diff
@@ -32,6 +32,9 @@ $testResultsDir = "$Root\artifacts\test\"
 $samplesDir = "$Root\src\Samples\Tests\Tests\"
 $ciDir = "$Root\ci\windows\"
 
+# set the codepage to UTF-8
+chcp 65001
```

Comment:
actually, is this file used now? it seems to use uitest.ps1

Reply:
Nope. It has been deleted.